buildSrc/src/main/groovy/org/elasticsearch/gradle/doc/RestTestsFromSnippetsTask.groovy

```diff
@@ -161,10 +161,25 @@ public class RestTestsFromSnippetsTask extends SnippetsTask {
             if (false == test.continued) {
                 current.println('---')
                 current.println("\"line_$test.start\":")
+                /* Since all these features *could* be required by any doc test
+                 * and we can't tell here if they *are* required by this doc
+                 * test we emit a skip all of them in case this is picked up by
+                 * a test runner that doesn't support them. */
```

Comment:
isn't the java runner the only one running these tests? I wonder why we need this